css/custom.css

```diff
@@ -45,7 +45,7 @@
 }
 
 html{
-	scroll-behavior: smooth;
+	/* scroll-behavior: smooth; */
```

Comment:
If this is no longer need, I would suggest to remove it as leaving it here could become confusing, if you think it really should stay could you add a comment to why that is?